app/src/app/components/input/input.component.scss

```diff
@@ -42,4 +42,14 @@ form {
             margin: 2px;
         }
     }
+    input[type=checkbox] {
```

Comment:
Why does github mark this red?

Are you sure this is syntactically correct?

Reply:
It is correct. this is a so called attribute selector. [Check this link.](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors)

Reply:
Okay. Was just wondering why this is highlighted red